tests/test_messages.py

```diff
@@ -382,6 +382,52 @@ def test_task_to_compute_validate_compute_task_def(self):
             ))
 
 
+class TasksTest(unittest.TestCase):
```

Comment:
Maybe put it in `test_tasks` like in https://github.com/golemfactory/golem-messages/pull/113/ ?

Reply:
@jiivan hmm... so maybe #113 should be merged first? ... but anyway, maybe we should add a subpackage `message` in `tests` and put those tests in there?

Reply:
good idea 👍